plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -23,6 +23,11 @@ def initialize(stop: false, live:, details:, pod:, role:, group:)
       end
     end
 
+    def self.build(output, job:, reference:)
```

Comment:
`*args` and then pass that on

Reply:
arg ... nah we need the job ... `output, job:. **args` maybe ... but not worth the trouble